.dlint.json

```diff
@@ -0,0 +1,9 @@
+{
+  "rules": {
+    "tags": ["recommended"],
+    "include": [
+      "ban-untagged-todo"
+    ],
+    "exclude": []
+  }
+}
```

Comment:
Seems like dlint’s default should be what we use in Deno - and therefore the config would be unnecessary.

fine with landing this as is now - but can  this config be removed in the futue?

Reply:
If we decide that `ban-untagged-todo` should be in the `recommended` set then sure.